### PR TITLE
Hellomicroverse

### PR DESCRIPTION
**Pull request by sheylaPozo**
Hello Dear Code Reviewer TSA, I would like to request a review for my project to can present it for Microverse's assignment #1 :) thank you in advance for your feedback!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ The main purpose of the project [Hello-Microverse](https://sheylapozo.github.io/
 
 ## Instructions
```
Comment (mricanho) [inline README.md:30]: - Good work on this section!

File: README.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ The main purpose of the project [Hello-Microverse](https://sheylapozo.github.io/
```
Comment (mricanho) [inline README.md:29]: - [ ] [OPTIONAL] Your live demo is broken, please fix it, or remove this section.

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @sheylaPozo , 

Welcome to Microverse and... good job so far! 🔥🔥 You follow the gitflow and almost all your README file is perfect. 

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

image
- [x] Your commit messages don't follow the convention, from now on try to write them in present-tense imperative style, instead of "linters", try "Add linters configuration", [this link]
(https://stackoverflow.com/questions/3580013/should-i-use-past-or-present-tense-in-git-commit-messages) explains why.
image
- [x] You have a good PR title, but the description must be that, a description of what did you do on this PR.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almost there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (sheylaPozo) [reply README.md:29]: fixed it

File: README.md (hunk 1 above)
Comment (sheylaPozo) [reply README.md:30]: thank you

**Comment by sheylaPozo** [comment]
Hello, Dear TSA, I would like if you could make a code review for me to approve this project, which I worked on with basic HTML and CSS, also worked with having a descriptive Read me file,  this project aims to practice acquiring the next...

**Learning Objectives**

- Create branches with different base branches.
- Understand the concept of version control.
- Follow Gitflow.
- Open pull requests in GitHub.
- Write descriptive, easy-to-understand commit messages.
- Write short, detailed, easy-to-understand descriptions of pull requests.
- Write a descriptive and easy-to-understand README in business English for every project submitted.
- Demonstrate an ability to submit a project for code review.

In this project, I set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices I've learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job as well, so it is important to know!

I've corrected the mistakes from the **code reviewer's suggestions** and requesting a new

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations @sheylaPozo :tada: :clinking_glasses: 

_Thank you for implementing the changes required by the last reviewer. Your commit messages are now better and your PR summary also looks great :smile:_

**Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:**

-------
**Comment**
- I'm dropping this suggestion so that you can do better in your next project. Your PR summary can be as simple as:
`
In this project, I set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices I've learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job as well, so it is important to know!
`
It should be a summary of the feature you've implemented, meaning that you do not have to use the entire project objectives. Kindly keep this in mind as you write a better PR summary in subsequent projects.

- - - - - -

**_Feel free to leave any questions or comments in the PR thread if something is not 100% clear._**

Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
Cheers, 
**[Taofeek Olalere](https://github.com/teekaytech)**
__Happy Coding :smiley:__

**Comment by sheylaPozo** [comment]
thank you for the feedback, I greatly appreciate it  :)

**Pull request merged**